spidermon/contrib/scrapy/monitors.py

```diff
@@ -0,0 +1,146 @@
+from spidermon import monitors, MonitorSuite, Monitor
+from spidermon.exceptions import NotConfigured
+from ..monitors.mixins.spider import SpiderMonitorMixin
+
+SPIDERMON_MIN_ITEMS = 'SPIDERMON_MIN_ITEMS'
+SPIDERMON_MAX_ERROR = 'SPIDERMON_MAX_ERROR'
```

Comment:
What about _SPIDERMON_MAX_ERROR**S**_ (plural)?

(MySQL uses _max_error_count_, but we could stick with the shorter version)